src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/CheatSheetSlimefunGuide.java

```diff
@@ -33,6 +39,19 @@ public boolean isSurvivalMode() {
         return false;
     }
 
+    @Override
+    protected List<Category> getVisibleCategories(Player p, PlayerProfile profile) {
```

Comment:
The annotations should be added here as well

Reply:
Ehm, you didn't actually add the annotations but added the javadocs instead?

Reply:
Oh crap XD